packages/runtime-common/loader.ts

```diff
@@ -892,3 +894,25 @@ function isEvaluatable(
   }
   return stateOrder[module.state] >= stateOrder['registered-completing-deps'];
 }
+
+async function maybeHandleScopedCSSRequest(req: Request) {
+  if (isScopedCSSRequest(req.url)) {
+    if (isFastBoot) {
+      return Promise.resolve(new Response('// skipped scoped CSS'));
```

Comment:
Is there something more suitable here? Or just `''`?

Reply:
maybe a 204 response (no content)?

Reply:
I don’t think anything is really needed, I changed it to 204, thanks 😀